packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -227,7 +265,7 @@ const Audio = ({
           isAction: true
         })
       }
-    } else if (track && track.isDelete) {
+    } else if (track && track.is_delete) {
       if (webRef.current) {
         // Sync w/ isPlaying false to set player state in dapp and hide drawer
         postMessage(webRef.current, {
```

Comment:
Are we keeping this to minimize breakages for now?

Reply:
just wanted to double check your meaning. the type previously was a track from the "syncQueue" state that i guess munges the track type to use camelCase for things, so this was typescript complaining since we are now using an actual track type.

Reply:
Sorry, this was unclear, I mean the `postMessage` back to the web view

Reply:
oh great call, can def remove those